test/test_functional.py

```diff
@@ -383,65 +350,6 @@ def test_pitch(self):
             s = ((freq - freq_ref).abs() > threshold).sum()
             self.assertFalse(s)
 
-            # Convert to stereo and batch for testing purposes
-            self._test_batch(F.detect_pitch_frequency, waveform, sample_rate)
-
-    def _test_batch_shape(self, functional, tensor, *args, **kwargs):
-
-        kwargs_compare = {}
-        if 'atol' in kwargs:
-            atol = kwargs['atol']
-            del kwargs['atol']
-            kwargs_compare['atol'] = atol
-
-        if 'rtol' in kwargs:
-            rtol = kwargs['rtol']
-            del kwargs['rtol']
-            kwargs_compare['rtol'] = rtol
-
-        # Single then transform then batch
-
-        torch.random.manual_seed(42)
-        expected = functional(tensor.clone(), *args, **kwargs)
-        expected = expected.unsqueeze(0).unsqueeze(0)
-
-        # 1-Batch then transform
-
-        tensors = tensor.unsqueeze(0).unsqueeze(0)
-
-        torch.random.manual_seed(42)
-        computed = functional(tensors.clone(), *args, **kwargs)
-
-        self._compare_estimate(computed, expected, **kwargs_compare)
-
-        return tensors, expected
-
-    def _test_batch(self, functional, tensor, *args, **kwargs):
-
-        tensors, expected = self._test_batch_shape(functional, tensor, *args, **kwargs)
-
-        kwargs_compare = {}
-        if 'atol' in kwargs:
-            atol = kwargs['atol']
-            del kwargs['atol']
-            kwargs_compare['atol'] = atol
-
-        if 'rtol' in kwargs:
-            rtol = kwargs['rtol']
-            del kwargs['rtol']
-            kwargs_compare['rtol'] = rtol
-
-        # 3-Batch then transform
-
-        ind = [3] + [1] * (int(tensors.dim()) - 1)
-        tensors = tensor.repeat(*ind)
-
-        ind = [3] + [1] * (int(expected.dim()) - 1)
-        expected = expected.repeat(*ind)
-
-        torch.random.manual_seed(42)
-        computed = functional(tensors.clone(), *args, **kwargs)
-
```

Comment:
@vincentqb As I go through this, I noticed that unlike the presumed intention of this test 3-batch test is not performed, so I removed it. If you think we should add 3-Batch test then I can do so either in a same PR or a separate one.

Reply:
Let's do this as a separate PR.